typescript/deploy/src/utils.ts

```diff
@@ -9,12 +9,13 @@ export function getArgs() {
   return yargs(process.argv.slice(2))
     .alias('e', 'env')
     .describe('e', 'deploy environment')
+    .string('e')
     .help('h')
     .alias('h', 'help');
 }
 
 export async function getEnvironment(): Promise<string> {
-  return (await getArgs().argv).e as Promise<string>;
+  return (await getArgs().argv).e as string;
```

Comment:
Like Nam said, this file is killed in my PR so I'll carry these changes over to my PR